### DISCUSSION TITLE
Welcome to FUDforum Discussions!

### DISCUSSION THREAD

**Discussion by naudefj** [fudforum/FUDforum#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by witcherre** [comment]
I'm "The Witcher" at Fudforum.org.  But it all seems dead over there.  Whats up?

**Reply by crtpc** [reply]
seems fudforum.org/forum is down. For a while it just tried to download a file [aka broken config] but now its just gone

**Reply by naudefj** [reply]
That's a surprise to me too!
Anyway, you can find everything you need to install and run the forum here on GitHub.

**Reply by redtropc** [reply]
yep, i set it up for my site because.... you never guessed it.... it works on ie6.
this forum functions entirely on Internet Explorer 6 and thats why i use it lol. plain ol http and browsin like its the 2000s. only corncern i have is that it hasnt been patched in almost a year [i hope the next yearly patch is soon] and that it runs eol php 7.4

**Comment by jethro-swan** [comment]
Can anyone here assist me in getting a current version of CKEditor working with the current version of FUDforum? FUDforum's plugin manager seems happy that CKEditor is installed, and I've installed and enabled CKEditor's required _bbcode_ and _entities_ plugins, but I can't see CKEditor's controls.

**Reply by jethro-swan** [reply]
(I realize that I've probably posted that in the wrong place.)

**Reply by naudefj** [reply]
CKEditor support was removed some time ago and replaced with an SCEditor plugin, which provides a much better WYSIWYG experience!

**Comment by gandsnut** [comment]
My intent is to use FF as a single-user, PRIVATE site.  Of course, I'm the admin.  When I sign in, I'm taken to the "Admin Control Panel".  Using 2 different browsers on my computer, how might I have the "Admin Control Panel" visible in one browser, and the FF board front page in the other browser?  Thx.

**Reply by naudefj** [reply]
Not sure I understand.
The forum and ACP have different URLs.
Maybe your forum is disabled and it’s routing you to the ACP?
